### PR TITLE
Use RHDocsBaseURL where possible

### PR DESCRIPTION
**Pull request by ekohl**
This is the result of runnning:

```
sed -i 's|https://access.redhat.com/documentation/en-us/|{RHDocsBaseURL}|' $(rg https://access.redhat.com/documentation/ guides/common/modules/ -l)
```

This should make it easier to switch to https://docs.redhat.com/

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Comment by asteflova** [comment]
It looks like you ended up with a double `en-US` in the new URLs. Can you try this instead?

```
sed -i 's|https://access.redhat.com/documentation/en-US/|{RHDocsBaseURL}|' $(rg https://access.redhat.com/documentation/ guides/common/modules/ -l)
```

**Comment by ekohl** [comment]
Oops, updated.

File: guides/common/modules/proc_using-insights-with-satellite-server.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ To maintain your {ProjectServer}, and improve your ability to monitor and diagno
 .Scheduling insights-client
 
 Note that you can change the default schedule for running `insights-client` by configuring `insights-client.timer` on {Project}.
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_insights/2020-10/html/client_configuration_guide_for_red_hat_insights/changing-the-client-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
+For more information, see {RHDocsBaseURL}red_hat_insights/2020-10/html/client_configuration_guide_for_red_hat_insights/changing-the-client-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
```
Comment (asteflova) [inline guides/common/modules/proc_using-insights-with-satellite-server.adoc:17]: This link gives a 404. I'm not familiar with Insights docs but I think this is the up-to-date one:
```suggestion
For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
```

Comment (ekohl) [reply guides/common/modules/proc_using-insights-with-satellite-server.adoc:17]: The exact file rename looks like a regression while the version makes sense. Is that something we should let the insights team know?

Comment (asteflova) [reply guides/common/modules/proc_using-insights-with-satellite-server.adoc:17]: Ok, I pinged the Insights docs team in an internal chat and shared a link to this comment thread. I don't think it should block this PR; if there is something to change on the Insights docs' side, they know where to start investigating.

**Comment by ekohl** [comment]
I noticed all pages redirect away from `en-us` so I added a commit that drops it. Waiting on the linkchecker to tell if that's correct.

File: guides/common/modules/proc_using-insights-with-satellite-server.adoc (hunk 1 above)
Comment (asteflova) [reply guides/common/modules/proc_using-insights-with-satellite-server.adoc:17]: The Insights docs team has confirmed that the suggested link is correct and follows their file naming guidelines: https://docs.redhat.com/en/documentation/red_hat_insights/1-latest/html-single/client_configuration_guide_for_red_hat_insights/index#assembly-client-changing-schedule

**Comment by ekohl** [comment]
If only Red Hat maintained stable documentation URLs ... but I think I have it now.

**Review by asteflova — APPROVED** [review]

**Comment by Lennonka** [comment]
Cherry-picked:
- 7902238063..eadc12316a  3.11 -> 3.11

It would be useful to do that in all mainted versions.

**Comment by ekohl** [comment]
The attribute commit itself wasn't picked so that's why I only checked 3.11, but I agree it makes sense

**Comment by ekohl** [comment]
@Lennonka I'd have preferred the commits weren't squashed. They were explicitly separated.

**Comment by Lennonka** [comment]
@ekohl Please, let us know next time in the PR description.

**Comment by ekohl** [comment]
Sorry, that's just how I always work so I didn't think about it

**Comment by Lennonka** [comment]
There's no need to apologize, but perhaps we should consider unifying the process, ie. clarifying when to squash and when not, how to label it, perhaps?

**Comment by ekohl** [comment]
I thought we settled that in https://github.com/theforeman/foreman-documentation/issues/884.

**Comment by Lennonka** [comment]
I wasn't even here when that discussion happened. How was I supposed to know? I still think we should revisit it. I'll put it on agenda for the next meeting.

**Comment by asteflova** [comment]
How about highlighting this in the PR template? https://github.com/theforeman/foreman-documentation/pull/3096

**Pull request merged**